styles.css

```diff
@@ -458,3 +469,414 @@ form {
 .message-label {
   padding-bottom: 40%;
 }
+
+.toolbar-links,
+.image-desktop,
+.about-project-desktop,
+.client-name-desktop,
+.name-desktop,
+.li-desktop,
+.project-desktop,
+.menu-logo {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
+  /* headliner page */
+  .toolbar {
+    width: 100vw;
+    height: 3.4vh;
+    position: relative;
+    justify-content: space-around;
+  }
+
+  .toolbar-links {
+    font-size: 0.6vw;
+    color: #344563;
+    text-decoration: none;
+    display: flex;
+    justify-content: start;
+    width: 10vw;
+    gap: 4%;
+  }
```

Comment:
- Just like the previous reviewer had mentioned, kindly remove the lines here as it is not looking the Figma design. Please see the below

![image](https://github.com/agneta1991/Portfolio/assets/89091822/3b93605b-a720-4589-ab53-d9c585d637dc)

Reply:
@Eternalgratis hi, i am really scratching my head here, am i missing something... as i am experimenting with different widths and it seams that background picture aligns well, but i thought that after the first review when submitting second time too, could you be kind and have a look at the screenshots included, as its different widths..

![different widths](https://github.com/agneta1991/Portfolio/assets/120168752/f2976516-a5b6-4efc-8a91-fa582e94ad6c)
![different widths2](https://github.com/agneta1991/Portfolio/assets/120168752/8b6cae7c-b87d-406c-b899-d0a17de20702)
![different widths3](https://github.com/agneta1991/Portfolio/assets/120168752/41743f92-294d-4aeb-a6d7-d0dc513fd24c)